frontend/encore/faq.rst

```diff
@@ -8,7 +8,7 @@ How do I deploy my Encore Assets?
 
 There are two important things to remember when deploying your assets.
 
-**1) Run ``encore production``**
+**1) Run "encore production"**
```

Comment:
What if we update this to be a title that explains in general what to do instead of what specific command to run. For example:

```rst
**1) Compile assets for production**
```